src/Components/Organisms/FOHeader/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { memo } from "react";
+import { AppBar, Toolbar, Button, IconButton, Typography } from "@mui/material";
```

Comment:
better to use component from atoms or molecules

Reply:
done